examples/README.md

```diff
@@ -1,11 +1,6 @@
-# Example SuperApps
+TODO...
```

Comment:
TODO

Reply:
will look into https://codesandbox.io - could be interesting to do for your redux sdk example